src/core/InstancedBufferGeometry.js

```diff
@@ -25,28 +25,26 @@ THREE.InstancedBufferGeometry.prototype.addDrawCall = function ( start, count, i
 
 	} );
 
-},
-
-THREE.InstancedBufferGeometry.prototype.clone = function () {
-
-	var geometry = new THREE.InstancedBufferGeometry();
+};
 
-	for ( var attr in this.attributes ) {
+THREE.InstancedBufferGeometry.prototype.copy = function ( source ) {
+	
+	for ( var attr in source.attributes ) {
 
-		var sourceAttr = this.attributes[ attr ];
+		var sourceAttr = source.attributes[ attr ];
 		geometry.addAttribute( attr, sourceAttr.clone() );
 
 	}
 
-	for ( var i = 0, il = this.drawcalls.length; i < il; i ++ ) {
+	for ( var i = 0, il = source.drawcalls.length; i < il; i ++ ) {
 
-		var offset = this.drawcalls[ i ];
+		var offset = source.drawcalls[ i ];
 		geometry.addDrawCall( offset.start, offset.count, offset.index, offset.instances );
```

Comment:
I think this should be `this.addDrawCall( offset.start, offset.count, offset.index, offset.instances );` instead.